app/src/main/java/io/pumpkinz/pumpkinreader/rest/ApiService.java

```diff
@@ -0,0 +1,19 @@
+package io.pumpkinz.pumpkinreader.rest;
```

Comment:
I think the package name is too specific. What if, we decided to make this realtime, and we don't use REST anymore.

I thing io.pumpkinz.pumpkinreader.service is more fitting.